Sources/ComposableArchitecture/SwiftUI/TextState.swift

```diff
@@ -68,6 +68,9 @@ public struct TextState: Equatable, Hashable {
     case strikethrough(active: Bool, color: Color?)
     case tracking(CGFloat)
     case underline(active: Bool, color: Color?)
+    case accessibilityLabel(AccessibilityLabelValue)
+    case accessibilityTextContentType(AccessibilityTextContentType)
+    case accessibilityHeading(AccessibilityHeadingLevel)
```

Comment:
Just a quick change for alphabetical order:
```suggestion
    case accessibilityHeading(AccessibilityHeadingLevel)
    case accessibilityTextContentType(AccessibilityTextContentType)
```
May move these to the beginning of the enum in the future, but should be good for now!